// python3 -m grpc_tools.protoc -I. --python_out=. --pyi_out=. --grpc_python_out=. multi.proto

syntax = "proto3";

message EndTurnRequest {}

message EndTurnResponse {
    string result = 1;
}

service GameService {
    rpc endTurn (EndTurnRequest) returns (EndTurnResponse);
}